test/e2e/scale_test.go

```diff
@@ -56,6 +70,43 @@ func TestE2ERetina_Scale(t *testing.T) {
 	chartPath := filepath.Join(rootDir, "deploy", "legacy", "manifests", "controller", "helm", "retina")
 	kubeConfigFilePath := filepath.Join(rootDir, "test", "e2e", "test.pem")
 
+	// Scale test parameters
+	opt := jobs.DefaultScaleTestOptions()
+	opt.KubeconfigPath = kubeConfigFilePath
+
+	NumDeployments := os.Getenv("NUM_DEPLOYMENTS")
+	NumReplicas := os.Getenv("NUM_REPLICAS")
+	NumNetworkPolicies := os.Getenv("NUM_NET_POL")
+	CleanUp := os.Getenv("CLEANUP")
+
+	if NumDeployments != "" {
+		opt.NumRealDeployments, err = strconv.Atoi(NumDeployments)
+		opt.NumRealServices = opt.NumRealDeployments
+		require.NoError(t, err)
+	}
+	if NumReplicas != "" {
+		opt.NumRealReplicas, err = strconv.Atoi(NumReplicas)
+		require.NoError(t, err)
+	}
+	if NumNetworkPolicies != "" {
+		opt.NumNetworkPolicies, err = strconv.Atoi(NumNetworkPolicies)
+		require.NoError(t, err)
+	}
+	if CleanUp != "" {
+		opt.DeleteLabels, err = strconv.ParseBool(CleanUp)
+		require.NoError(t, err)
+	}
+
+	RetinaVersion := os.Getenv(generic.DefaultTagEnv)
+	require.NotEmpty(t, RetinaVersion)
+	opt.AdditionalTelemetryProperty["retinaVersion"] = RetinaVersion
+	opt.AdditionalTelemetryProperty["clusterName"] = clusterName
+
+	// AppInsightsKey is required for telemetry
+	require.NotEmpty(t, os.Getenv("APP_INSIGHTS_KEY"))
+
+	opt.LabelsToGetMetrics = map[string]string{"k8s-app": "retina"}
+
 	// CreateTestInfra
 	createTestInfra := types.NewRunner(t, jobs.CreateTestInfra(subID, rg, clusterName, location, kubeConfigFilePath, *createInfra))
```

Comment:
This part is not yet agreed. Right now this pipeline execution is not automatically triggered by any event, only manual execution, so a cluster can be provided to run the test.

I will delete Cluster creation for now to avoid any confusion since this is not being used.